### PR TITLE
feat(safer): validate files with safer

### PR DESCRIPTION
**Pull request by maljuburi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "example": "./scripts/example.sh",
     "test": "jest",
     "lint": "eslint",
-    "prepare": "corepack prepare pnpm@latest --activate && pnpm build:ipfs && pnpm build:browser && pnpm build:node && cp dist/index.d.ts dist/browser/index.d.ts",
+    "prepare": "corepack prepare pnpm@7.9.4 --activate && pnpm build:ipfs && pnpm build:browser && pnpm build:node && cp dist/index.d.ts dist/browser/index.d.ts",
```
Comment (aewing) [inline package.json:47]: we want latest

Comment (aewing) [reply package.json:47]: if @latest doesn't work for you, update your node version to 16.17+

Comment (maljuburi) [reply package.json:47]: Yup, latest wasn't working for me.. I am on node `16.17.0`. maybe I should bump my patch higher than `0`  
Screen Shot 2022-08-29 at 2 00 22 PM

Comment (maljuburi) [reply package.json]: Fixed! I had two global `pnpm` and was causing problems. all good now. Thanks Drew

**Review by aewing — APPROVED** [review]

**Pull request merged**